.travis.yml

```diff
@@ -25,7 +25,9 @@ before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash
   - export PATH=$HOME/.yarn/bin:$PATH
 
-install: yarn --frozen-lockfile
+install:
+  - yarn --frozen-lockfile
+  - yarn run setup
```

Comment:
This should be a `postinstall` lifecycle, not a separate call?

Reply:
Using a postinstallhere currently is blocked by #1125

Reply:
Hah, of course, you're right. I've been trying (and largely failing) to make tests reasonable for that one.